sdk/trace/src/main/java/io/opentelemetry/sdk/trace/SpanLimitsBuilder.java

```diff
@@ -10,6 +10,9 @@
 
 /** Builder for {@link SpanLimits}. */
 public final class SpanLimitsBuilder {
+
+  // These values are the default values for all the limits.
```

Comment:
does this even add any value since the fields have the word DEFAULT in them already?